pkgs/applications/science/biology/snpeff/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "snpeff-${version}";
-  version = "4.3q";
+  version = "4.3t";
 
   src = fetchurl {
-    url = "mirror://sourceforge/project/snpeff/snpEff_v4_3q_core.zip";
-    sha256 = "0sxz8zy8wrzcy01hyb1cirwbxqyjw30a2x3q6p4l7zmw2szi7mn1";
+    url = "mirror://sourceforge/project/snpeff/snpEff_v4_3t_core.zip";
```

This would enable the auto updater to bump the version next time:
```suggestion
    url = "mirror://sourceforge/project/snpeff/snpEff_${builtins.replaceStrings [ "." ] [ "_" ] $version}_core.zip";
```

---

Great idea, thanks!

---

Looks like there's a 'v' missing.
```suggestion
    url = "mirror://sourceforge/project/snpeff/snpEff_v${builtins.replaceStrings [ "." ] [ "_" ] $version}_core.zip";
```

---

Thanks, didn't catch that because of caching.